src/scenes/AcceleratorRouter/Voting/VotingEdit.tsx

```diff
@@ -70,6 +70,7 @@ const VotingEdit = () => {
     if (result?.status === 'error') {
       snackbar.toastError();
     } else if (result?.status === 'success') {
+      reloadVotes?.();
```

Comment:
Instead of plumbing this through the component, can you reload the votes within the `requestProjectVotesUpdate` thunk? The second argument contains `dispatch` which you can use to dispatch the get request from within the thunk, if the request is successful.

```export const requestProjectVotesUpdate = createAsyncThunk(
  'votes/update',
  async (request: { projectId: number; payload: UpsertProjectVotesRequestPayload }, { rejectWithValue }) => {```
->
```
export const requestProjectVotesUpdate = createAsyncThunk(
  'votes/update',
  async (request: { projectId: number; payload: UpsertProjectVotesRequestPayload }, { dispatch, rejectWithValue }) => {
```

```
    if (response.requestSucceeded) {
      return response?.data?.status === 'ok' ? true : false;
    }
```
->
```
    if (response.requestSucceeded) {
      dispatch(requestProjectVotesGet(request.projectId));
      return response?.data?.status === 'ok' ? true : false;
    }
```

Reply:
This makes a lot of sense! Thank you.

Reply:
Thanks for this discussion. This is fine.
Thoughts on API requests being more granular and have the UX handle the intent of refetching data?